lib/plot/falco_plot_progress_dst.m

```diff
@@ -218,35 +222,46 @@
     tag = '';
 end
 
-sciCam_fitswrite(tb,Im,fullfile(out_dir,['normI_it',num2str(Itr-1),tag,'.fits']));
+fitswrite_tb(mp,tb,Im,fullfile(out_dir,['normI_it',num2str(Itr-1),tag,'.fits']));
 
 if(any(mp.dm_ind==1) && Itr==1)
-    sciCam_fitswrite(tb,mp.dm1.biasMap,fullfile(out_dir,'dm1_Vbias.fits'));
+    fitswrite_tb(mp,tb,mp.dm1.biasMap,fullfile(out_dir,'dm1_Vbias.fits'));
 end
 if(any(mp.dm_ind==2) && Itr==1)
-    sciCam_fitswrite(tb,mp.dm2.biasMap,fullfile(out_dir,'dm2_Vbias.fits'));
+    fitswrite_tb(mp,tb,mp.dm2.biasMap,fullfile(out_dir,'dm2_Vbias.fits'));
 end
 
 if(any(mp.dm_ind==1))
-    sciCam_fitswrite(tb,mp.dm1.V,fullfile(out_dir,['dm1_V_it',num2str(Itr-1),tag,'.fits']));
-    sciCam_fitswrite(tb,DM1surf,fullfile(out_dir,['dm1_model_it',num2str(Itr-1),tag,'.fits']));
+    fitswrite_tb(mp,tb,mp.dm1.V,fullfile(out_dir,['dm1_V_it',num2str(Itr-1),tag,'.fits']));
+    fitswrite_tb(mp,tb,DM1surf,fullfile(out_dir,['dm1_model_it',num2str(Itr-1),tag,'.fits']));
 end
 if(any(mp.dm_ind==2))
-    sciCam_fitswrite(tb,mp.dm2.V,fullfile(out_dir,['dm2_V_it',num2str(Itr-1),tag,'.fits']));
-    sciCam_fitswrite(tb,DM2surf,fullfile(out_dir,['dm2_model_it',num2str(Itr-1),tag,'.fits']));
+    fitswrite_tb(mp,tb,mp.dm2.V,fullfile(out_dir,['dm2_V_it',num2str(Itr-1),tag,'.fits']));
+    fitswrite_tb(mp,tb,DM2surf,fullfile(out_dir,['dm2_model_it',num2str(Itr-1),tag,'.fits']));
 end
 
 
-sciCam_fitswrite(tb,abs(Im_tb.E).^2,fullfile(out_dir,['normI_Esens_it',num2str(Itr-1),tag,'.fits']));
-sciCam_fitswrite(tb,angle(Im_tb.E),fullfile(out_dir,['phz_Esens_it',num2str(Itr-1),tag,'.fits']));
-sciCam_fitswrite(tb,Im_tb.Iinco,fullfile(out_dir,['normI_inco_it',num2str(Itr-1),tag,'.fits']));
+fitswrite_tb(mp,tb,abs(Im_tb.E).^2,fullfile(out_dir,['normI_Esens_it',num2str(Itr-1),tag,'.fits']));
+fitswrite_tb(mp,tb,angle(Im_tb.E),fullfile(out_dir,['phz_Esens_it',num2str(Itr-1),tag,'.fits']));
+fitswrite_tb(mp,tb,Im_tb.Iinco,fullfile(out_dir,['normI_inco_it',num2str(Itr-1),tag,'.fits']));
 
 if(~strcmpi(mp.estimator,'perfect'))
     ev = Im_tb.ev;
     save(fullfile(out_dir,['probing_data_',num2str(Itr-1),tag,'.mat']),'ev');
 end
-
+if strcmpi(mp.estimator,'ekf_maintenance') && any(mp.est.itr_ol==ev.Itr) == true
+    img = mean(Im_tb.ev.normI_OL_sbp,3);
+    fitswrite_tb(mp,tb,img,fullfile(out_dir,['normI_OL_it',num2str(Itr-1),tag,'.fits']));
+end
 % Update the diary 
 diary off; diary(mp.diaryfile)
 
 end %--END OF FUNCTION
+
+function fitswrite_tb(mp, tb, obj, filename)
+if mp.flagSim
+    fitswrite(obj,filename);
+else
+    sciCam_fitswrite(tb,obj,filename);
+end
+end
```

Comment:
added this function because sciCam fitswrite doesn't work in simulation.  I am happy to change this as long is the change will run in simulation.  All plot_progress_testbed files need a function like this so maybe this small function should live somewhere else to reduce duplicated code?